src/__tests__/01.basic.ts

```diff
@@ -37,6 +29,14 @@ export interface TestSchema {
    * propertyName1
    */
   propertyName1: boolean;
+  /**
+   * dateCreated
+   */
+  dateCreated?: Date;
```

Comment:
are you able to add a property sort? this will keep these results the same

Reply:
Yea i can add it. You cool if i leave it as setting? I think it follows the order defined on the joi schema which i like personally.

Reply:
setting sounds good
The reason for it is in a previous code generation project I had issues with properties generating in the incorrect order